Source/VCGeneration/VCGen.cs

```diff
@@ -356,18 +366,26 @@ void ExpandAsserts(Implementation impl)
       watch.Start();
 #endif
 
-      ConvertCFG2DAG(run.Implementation);
+      var data = implementationData.GetOrCreateValue(run.Implementation)!;
+      if (!data.ConvertedToDAG) {
+        data.ConvertedToDAG = true;
```

Comment:
I don't know how much it matters, but this sets `convertedToDAG` to `true` before converting it, whereas `Passified`, below, is the other way around.